include/zephyr/bluetooth/gap.h

```diff
@@ -26,6 +26,15 @@ extern "C" {
  * @{
  */
 
+/**
+ * @brief Default GAP service name
+ *
+ * Use this name as a first argument for BT_GATT_SERVICE_DEFINE when creating
+ * GAP service implementation.
+ * This ensures that the GAP service would be placed just after BT_UUID_GATT.
+ */
+#define BT_GATT_GAP_SVC_DEFAULT_NAME _2_gap_svc
```

Comment:
This should either be defined in `gatt.h`, or renamed to 
```suggestion
#define BT_GAP_GATT_SVC_DEFAULT_NAME _2_gap_svc
```

Is it a requirement that GAP service is placed there in the database? If not, should this be a Kconfig option?